ecosystem/resources/members/openqasm.toml

```
@@ -1,6 +1,6 @@
 name = "OpenQASM"
 url = "https://github.com/openqasm/openqasm"
-description = "OpenQASM is an imperative programming language designed for near-term quantum computing algorithms and applications. Quantum programs are described using the measurement-based quantum circuit model with support for classical feed-forward flow control based on measurement outcomes."
+description = "An imperative programming language for near-term quantum computing. Describe quantum programs by using the measurement-based quantum circuit model with support for classical feed-forward flow control based on measurement outcomes."
```

```suggestion
description = "An imperative programming language for near-term quantum computing. Describe quantum programs by using the measurement-based quantum circuit model with support for classical feedforward flow control based on measurement outcomes."
```
removing hyphen based on how Kevin Sung uses feedforward in the docs he wrote